src/gobmessage/hr/kvk/dataservice/service.py

```diff
@@ -41,6 +42,30 @@ def _generate_reference(self):
         dt = datetime.utcnow()
         return f"GOB-{dt.strftime('%Y%m%d%H%M%S%f')}"
 
+    def _unpack_raw_elements(self, in_dict: dict):
+        """Unpacks _raw_elements dicts in Zeep result. _raw_elements is a list of elements that aren't parsed by Zeep
+        because the result does not conform to the XSD. We do this manually in this method.
+
+        :param in_dict:
+        :return:
+        """
+        result = {}
+
+        if '_raw_elements' in in_dict:
+            # Return all existing elements in dict, updated with the values in _raw_elements.
+            return {k: self._unpack_raw_elements(v) if isinstance(v, dict) else v
+                    for k, v in in_dict.items() if k != '_raw_elements'
+                    } | {re.sub(r'^{.*}', '', elm.tag): elm.text for elm in in_dict['_raw_elements']}
+
+        for k, v in in_dict.items():
+            if isinstance(v, dict):
+                result[k] = self._unpack_raw_elements(v)
+            elif isinstance(v, list) and len(v) > 0 and isinstance(v[0], dict):
```

Comment:
Misschien beter all(isinstance(d, dict) for d in v) i.p.v. alleen de check op het eerste element in de lijst.